includes/class-admin-settings.php

```diff
@@ -49,6 +49,7 @@ public function __construct() {
 		add_action( 'admin_init', [ $this, 'reset_settings' ] );
 		add_action( 'admin_init', [ $this, 'register_settings' ] );
 		add_action( 'admin_notices', [ $this, 'reset_admin_notice' ] );
+		add_action( 'admin_notices', [ $this, 'settings_saved_admin_notice' ] );
```

Comment:
The hook name should be conditional on `is_multisite() ? 'network_admin_notices' : 'admin_notices'`